web/test/pages/roadmap.test.tsx

```diff
@@ -108,7 +102,7 @@ describe("roadmap page", () => {
     return { page };
   };
 
-  const renderPageWithAuth = ({
+  const renderPageWithAuthAlert = ({
```

Comment:
why this rename?

Reply:
This function was using the `withAuthAlert` helper function and rendered the Roadmap page with SignUp Toast component. I didn't need the SignUp Toast component to render so I created a separate function called `renderPageWithAuth` and used the `withAuth` helper function. This function was renamed because it shared the same name.